lib/resources/example_json/dstu2_examples_medication_administration.rb

```diff
@@ -0,0 +1,378 @@
+              "div": "<div><p><b>Medication Administration</b></p><p><b>Patient Name</b>: ROEN, LESLEY MICHELE</p><p><b>Medication Display</b>: ethinyl estradiol-norgestrel (ethinyl estradiol-norgestrel 30 mcg-0.3 mg oral tablet)</p><p><b>Was Not Given</b>: False</p><p><b>Dosage Quantity</b>: 1</p><p><b>Medication Status</b>: in-progress</p><p><b>Effective Time</b>: 2009-04-15T15:58:00.000Z</p></div>"
+            },
+            "status": "in-progress",
+            "patient": {
+              "reference": "Patient/1061997",
+              "display": "ROEN, LESLEY MICHELE"
+            },
+            "practitioner": {
+              "reference": "Practitioner/607928",
+              "display": "Fox, Jason"
+            },
+            "encounter": {
+              "reference": "Encounter/1113890"
+            },
+            "prescription": {
+              "reference": "MedicationOrder/9554880"
+            },
+            "effectiveTimeDateTime": "2009-04-15T15:58:00.000Z",
+            "medicationCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                  "code": "311359",
+                  "display": "Ethinyl Estradiol 0.03 MG / Norgestrel 0.3 MG Oral Tablet",
+                  "userSelected": false
+                }
+              ],
+              "text": "ethinyl estradiol-norgestrel (ethinyl estradiol-norgestrel 30 mcg-0.3 mg oral tablet)"
+            },
+            "dosage": {
+              "text": "1 tab(s) Oral Daily",
+              "route": {
+                "coding": [
+                  {
+                    "system": "http://ncimeta.nci.nih.gov",
+                    "code": "C38288",
+                    "display": "ORAL",
+                    "userSelected": false
+                  }
+                ],
+                "text": "ORAL"
+              },
+              "quantity": {
+                "value": 1,
+                "unit": "{tbl}",
+                "system": "http://unitsofmeasure.org"
+              }
+            }
+          }
+        },
+        {
+          "fullUrl": "http://localhost:3002/dstu2/2c400054-42d8-4e74-87b7-80b5bd5fde9f/MedicationAdministration/2008538",
+          "resource": {
+            "resourceType": "MedicationAdministration",
+            "id": "2008538",
+            "meta": {
+              "versionId": "2008539",
+              "lastUpdated": "2009-04-15T15:59:03.000Z"
+            },
+            "text": {
+              "status": "generated",
+              "div": "<div><p><b>Medication Administration</b></p><p><b>Patient Name</b>: ROEN, LESLEY MICHELE</p><p><b>Medication Display</b>: fluticasone-salmeterol (fluticasone-salmeterol 250 mcg-50 mcg inhalation powder)</p><p><b>Was Not Given</b>: True</p><p><b>Medication Status</b>: stopped</p><p><b>Effective Time</b>: 2009-04-15T14:00:00.000Z</p></div>"
+            },
+            "status": "stopped",
```

Comment:
these statuses are a bit surprising based on what I know of our system. Maybe we need to double-check our mappings? I would assume most would come out as completed or entered-in-error. I could see stopped if we somehow differentiated between not given and partially given?

Reply:
(note: maybe stopped is what we use whenever it's not done/given. But in-progress seems surprising)

Reply:
I think we discussed this offline. Ultimately the available statuses should be Completed, Stopped and Entered-In-Error. There was some discussion about this on the mappings Jira after we discovered that in-progress was being returned. I'm currently working on getting those corrected mappings pushed out.